spec/unit/shoulda/matchers/active_model/validate_numericality_of_matcher_spec.rb

```diff
@@ -2080,7 +2226,11 @@ def apply_qualifiers!(args)
 
     combination.each do |qualifier|
       args = self.class.default_qualifier_arguments.fetch(qualifier[:name])
-      matcher.__send__(qualifier[:name], *args)
+      if args
+        matcher.__send__(qualifier[:name], args)
+      else
+        matcher.__send__(qualifier[:name])
+      end
```

Comment:
This method was causing an error because `args` when testing the `in` option was a range, and the splat operator was destructuring the range.

We were using the splat operator on `args` only to prevent passing it when it was nil, so I changed this method to reflect that better and not cause the error when using a range.

Happy to change to this other implementation if you think the use case of not destructuring is more clear when the argument is a range.
```suggestion
      if args.is_a? Range
        matcher.__send__(qualifier[:name], args)
      else
        matcher.__send__(qualifier[:name], *args)
      end
```